src/main/java/seedu/duke/Duke.java

```diff
@@ -3,19 +3,20 @@
 import java.util.Scanner;
 
 public class Duke {
-    /**
-     * Main entry-point for the java.duke.Duke application.
-     */
     public static void main(String[] args) {
```

Comment:
Would it better to refactor all the key functions out of the 'main' method?

Reply:
Alright